### PR TITLE
prep-javascript-objects

### PR DESCRIPTION
**Pull request by bug-smith**
Screenshot 2023-04-19 at 2 17 57 PM

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const person = {
+  firstName: 'Bug',
+  lastName: 'Smith',
+  hobby: 'golfing',
+  job: 'unemployed',
+  previousJob: 'Techstack Admin',
```
Comment (BrettAlbright) [inline prep-javascript-objects/main.js]: the job property should be added to the object using dot notation. 
the previous job property should be added to the object using bracket notation.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates, watch for those details in the instructions!

**Comment by bug-smith** [comment]
Screenshot 2023-04-19 at 4 51 52 PM

Hi Brett,

ahh yes, i think i was confused and was trying to come up with another way to get the same result.

I think i have updated the code to be correct, utilizing nesting within the variable for my objects, and updating the current job to use dot notation to request from the variable/object tree.

Thank you

**Review by Zhiwutian — APPROVED** [review]
That was an interesting way to go!

**Pull request merged**